src/test/scala/com/github/sbt/sbom/BomExtractorSpec.scala

```diff
@@ -0,0 +1,55 @@
+package com.github.sbt.sbom
+
+import org.scalatest.featurespec.AnyFeatureSpecLike
+import org.scalatest.GivenWhenThen
+
+class BomExtractorSpec extends AnyFeatureSpecLike with GivenWhenThen {
```

Comment:
So far we've been using the `AnyWordSpec with Matchers` style - any strong reason to deviate from that?